EpiAware/docs/src/developer/contributing.md

```diff
@@ -39,3 +39,25 @@ As is conventional for Julia packages, unit tests are located at `test/*.jl` wit
 Tests that build example package docs from source and inspect the results (end to end tests) are
 located in `/test/examples`. The main entry points are `test/examples/make.jl` for building and
 `test/examples/test.jl` for doing some basic checks on the generated outputs.
+
+### Pluto usage in end to end tests and showcases
+Some of the end to end tests and showcases use `Pluto.jl` scripts.
```

Comment:
Is this a contribution approach we want to support? Or is this where we say to contribute: 

1. Clone the repo
2. Run the pluto server script we already have
3. Make change to pluto doc? 

i.e do we really need to support this workflow? 

(I also think the pluto in tests are a legacy that we need to remove).

Reply:
I think having the Pluto in the showcases makes sense, especially now that we have an ok handle on making it easier to use. But I'm removing reference to testing.